build/helper/documentation_snippets.py

```diff
@@ -96,6 +96,14 @@
 { 'simulate': True, 'driver_setup': { 'Model': '<model number>',  'BoardType': '<type>' } }
 '''
 
+default_close_function_doc = '''
+Closes the driver session and cleans up.
+'''
+
+close_function_note = '''
+This function is not needed when using the session context manager
+'''
+
 default_initiate_function_doc = '''
 Calls initiate
```

Comment:
> "`Calls initiate`"
Is this the documentation we'd get for `initiate()`?
It's useless, no?

Reply:
This is only used if initiate doesn't have its own documentation. In theory, you should never see this. Given that, I am open to a better, generic description.

Reply:
> This is only used if initiate doesn't have its own documentation.

I'd rather have this be a build error.

Reply:
You want to require there be supplied documentation? Is that reasonable?